src/config/unit_test/mod.rs

```diff
@@ -557,6 +562,31 @@ fn build_input_event(input: &TestInput) -> Result<Event, String> {
             Some(v) => Ok(Event::Log(LogEvent::from_str_legacy(v.clone()))),
             None => Err("input type 'raw' requires the field 'value'".to_string()),
         },
+        "vrl" => {
+            if let Some(source) = &input.source {
+                let fns = vrl::stdlib::all();
+                let result = match vrl::compiler::compile(source, &fns) {
+                    Ok(v) => v,
+                    Err(e) => return Err(Formatter::new(source, e.clone()).to_string()),
+                };
```

Comment:
Using `map_err` makes this a little more concise:
```suggestion
                let result = vrl::compiler::compile(source, &fns)
                    .map_err(|e| Formatter::new(source, e.clone()).to_string())?;
```

Reply:
Did that now! Thanks